pkg/build/package.go

```diff
@@ -114,7 +114,8 @@ func (pc *PackageContext) EmitPackage() error {
 		tarball.WithOverrideUIDGID(0, 0),
 		tarball.WithOverrideUname("root"),
 		tarball.WithOverrideGname("root"),
-		tarball.WithUseChecksums(true),
+		//FIXME: not used anymore ??
```

Comment:
This feature needs to be enabled, it attaches the *expected* checksum of each file in the tarball as a PAX record.  apk-tools *absolutely* uses this when using APKv2 format packages.

Reply:
It's just that the function doesn't exist https://github.com/chainguard-dev/melange/search?q=WithUseChecksums. Maybe it's still in a local diff ?

Reply:
https://github.com/chainguard-dev/apko/blob/main/pkg/tarball/tarball.go#L100-L107

Reply:
Oh sorry 🤦🏼